sdk/mobile-center-push/src/test/java/com/microsoft/azure/mobile/push/PushTest.java

```diff
@@ -177,13 +187,21 @@ public void setEnabled() {
         verify(channel).enqueue(any(PushInstallationLog.class), eq(push.getGroupName()));
 
         /* Disable. */
-        Push.setEnabled(false);
+        Push.setEnabled(false).get();
```

Comment:
it's just additional testing to make sure there is no deadlock or any issue by doing that as well.